src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        this.head = null;
+        this.tail = null;
+        this.size = 0;
+    }
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value);
+        if (tail == null) {
+            // Якщо список порожній
+            head = newNode;
+            tail = newNode;
+        } else {
+            // Додаємо новий вузол в кінець списку
```

Comment:
Don't begin method implementation with an empty line. Remove the redundant empty line at the start of the block.